src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #f3f3f3;
+}
+
+.card {
+  max-width: 198px;
```

Comment:
should be 200px (with `box-sizing: border-box;` width includes borders as well)
check requirements

Reply:
add a border as in the mockup